client/landing/jetpack-cloud/sections/backups/main.js

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
I think this PR is doing far too many things. Can we separate the framework changes from the component ones?

Reply:
Yes that can be done.